XPlat.ApplicationModel/Package.iOS.cs

```diff
@@ -0,0 +1,88 @@
+﻿#if __IOS__
+namespace XPlat.ApplicationModel
+{
+    using System;
+    using System.Collections.Generic;
+
+    using global::Foundation;
+
+    using XPlat.Storage;
+
+    /// <summary>Provides information about a package.</summary>
+    public class Package : IPackage, IPackage2, IPackage3
+    {
+        private static Package current;
+
+        private readonly WeakReference originatorReference;
+
+        private IStorageFolder installedLocation;
+
+        private IPackageId id;
+
+        public Package(NSBundle bundle)
+        {
+            if (bundle == null)
+            {
+                throw new ArgumentNullException(nameof(bundle));
+            }
+
+            this.originatorReference = new WeakReference(bundle);
+        }
+
+        /// <summary>Gets the package for the current app.</summary>
+        public static Package Current => current != null && current.originatorReference.IsAlive ? current : (current = NSBundle.MainBundle);
+
+        /// <summary>Gets the package identity of the current package.</summary>
+        public IPackageId Id => this.id ?? (this.id = new PackageId(this.Originator));
+
+        /// <summary>Gets the location of the installed package.</summary>
+        public IStorageFolder InstalledLocation => this.installedLocation ?? (this.installedLocation = new StorageFolder(this.Originator.BundlePath));
+
+        /// <summary>Gets the packages on which the current package depends.</summary>
+        public IReadOnlyList<IPackage> Dependencies => new List<IPackage>();
+
+        /// <summary>Gets the display name of the package.</summary>
+        public string DisplayName => this.Originator?.ObjectForInfoDictionary("CFBundleDisplayName")?.ToString();
+
+        /// <summary>Gets the logo of the package.</summary>
+        public Uri Logo => null;
```

Comment:
Why doesn't this return anything?

Reply:
So I couldn't work out or find much around documentation on how to get the path to the logo. I thought I would leave this, for now, unimplemented until potentially I find a result on this.

Are you happy with that?

Reply:
That's not a problem, as long as this is documented

Reply:
I will add some text in the GitBooks docs for this 👍

Reply:
Awesome, thanks!